Microsoft.Toolkit.Services/PlatformSpecific/NETFramework/PopupForm.cs

```diff
@@ -46,6 +44,8 @@ private void WebViewNavigationStartingHandler(Uri uri)
             if (initialHost != topLevelDomain && topLevelDomain == callbackHost)
             {
                 ActualUrl = uri;
+                webView1.Dispose();
+                webView1.Refresh();
```

Comment:
Why `Refresh` after `Dispose`?

Reply:
We were trying out something with the WebView that didn't work out at the end, and forgot to remove it. We'll submit a commit to do so.